cmd/vulcan-aws-alerts/rds.go

```diff
@@ -0,0 +1,101 @@
+package main
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/adevinta/vulcan-report"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/rds"
+
+	"github.com/adevinta/vulcan-check-sdk/state"
+)
+
+func caCertificateRotation(target string, vulcanAssumeRoleEndpoint string, roleName string, state state.State) error {
+	sess, err := session.NewSession(&aws.Config{})
+	if err != nil {
+		return err
+	}
+
+	creds, err := getCredentials(vulcanAssumeRoleEndpoint, target, roleName)
+	if err != nil {
+		return err
+	}
+
+	// Iterate over all AWS regions where RDS is available
+	for region := range endpoints.AwsPartition().Services()[endpoints.RdsServiceID].Regions() {
+		sess.Config.Region = aws.String(region)
+		s := rds.New(sess, &aws.Config{Credentials: creds})
+
+		ctx := context.Background()
+		logger.Info(fmt.Sprintf("Describing pending maintenance actions on RDS for %s region", region))
+		result, err := s.DescribePendingMaintenanceActionsWithContext(ctx, &rds.DescribePendingMaintenanceActionsInput{})
+		if err != nil {
+			logger.Error(err)
+			continue
+		}
+		// TODO: implement pagination
+		if result.Marker != nil {
+			logger.Warn("DescribePendingMaintenanceActionsWithContext returned more than 1 page")
+		}
+
+		rg := report.ResourcesGroup{
+			Name:   `Instances`,
+			Header: []string{"identifier", "account", "region", "dbname", "engine", "arn", "AutoAppliedAfterDate", "CurrentApplyDate"},
```

Comment:
Suggestion: use `Identifier`, `Account`, etc. instead of `identifier`, etc.

Reply:
Done!